### ISSUE TITLE
May I ask how to mitigate over-saturation and plastic-face issue?

### ISSUE THREAD

**Issue by Shilin-LU** [bytedance/DreamO#83]
Thank you for your amazing work on this project! It’s been incredibly helpful and impressive.

If you don’t mind, could you please share a few more details on **how over-saturation and plastic-face problems were mitigated?** I’m particularly interested in understanding:
- What methods or techniques were most effective?
- Whether the solution involved adjustments to the model, loss, training data, or post-processing?

I’d really appreciate any insights you can share. Thanks again for your efforts and support!

**Comment by Shilin-LU** [comment]
Thank you for your response! I noticed that the vanilla Flux doesn’t seem to have significant issues with plasticity, but when using other adapters, this problem tends to occur more frequently. Do you have any insights into why that might be?

**Comment by ToTheBeginning** [comment]
Possible reasons include the following:

1. The datasets used for training most adapters are generally of lower quality than those used for training the base models. In particular, constructing high-quality paired data is much more challenging than constructing high-quality T2I data.
2. The more parameters of the base model are trained (full fine-tuning > LoRA > ipadapter-like architectures), the more susceptible the model becomes to the quality of the training dataset, leading to potential quality drift.

**Comment by Shilin-LU** [comment]
@ToTheBeginning Thank you for your insights! It makes sense.

**Comment by yash2code** [comment]
is there any quick workaround to solve plasticity ?
